src/main/java/com/dev/cinema/controller/HelloController.java

```diff
@@ -1,10 +1,10 @@
 package com.dev.cinema.controller;
 
-import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
 
-@Controller
+@RestController
 public class HelloController {
     @ResponseBody
```

Comment:
do we need this annotation now? What is the difference between @Controller and @RestController?

Reply:
sorry was inattentive
@RestController already contains @ResponseBody in it